packages/pigment-css-react/src/useTheme.d.ts

```diff
@@ -1,3 +1,3 @@
 import { ThemeArgs } from './theme';
 
-export default function useTheme(): ThemeArgs extends { theme: any } ? ThemeArgs['theme'] : any;
+export default function useTheme(): ThemeArgs extends { theme: unknown } ? ThemeArgs['theme'] : any;
```

Comment:
```suggestion
export default function useTheme(): ThemeArgs['theme'] extends any ? ThemeArgs['theme'] : any;
```

Reply:
This is an alternative that also works, this may be more readable. What do you think @brijeshb42?